build/yaml/crd/nsx.vmware.com_ippools.yaml

```diff
@@ -183,10 +183,12 @@ spec:
                   type: object
                 type: array
               type:
-                description: Type defines the type of this IPPool, Public or Private.
+                description: Type defines the type of this IPPool, can be Public,
+                  Private, PrivateTGW.
                 enum:
                 - Public
                 - Private
+                - PrivateTGW
```

Comment:
Should we use `Private_tgw` as it's NSX VPC API type?

Reply:
Private_TGW

Reply:
I feel let us follow Golang convention for now (that is no "_").